src/MarginTieBreaker.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace PivotLibre\Tideman;
+
+use PivotLibre\Tideman\ListOfMarginLists;
+
+/**
+ * A MarginTieBreaker specifies some tie-breaking rule that determines which Margin should be considered the winner
+ * over another Margin even though their difference properties are identical.
+ */
+interface MarginTieBreaker
+{
+
+    /**
```

Comment:
Crap, this interface's description, and all the implementations have the wrong sign convention. It got confused because "more preferred" means "lower index". I will update them soon.

Reply:
Fixed.